File: Muneef khan 43 (A-1)/C++/Q4.cpp
//Q-4) Write a C++ program to sort a given array of 0s, 1s and 2s. In the final array put all 0s first, then all 1s and all 2s in last.

#include<iostream>
using namespace std;
int main()
{
    int n;
    cout<<"Enter size of an array: ";
    cin>>n;
    int a[n];
    cout<<"Enter "<<n<<" number of 0s, 1s and 2s: ";
    for (int i = 0; i < n; i++)
    {
        cin>>a[i];
    }
    
    for (int i = 0; i < n-1; i++)
    {
        for (int j = i+1; j < n; j++)
        {
            if(a[i]>a[j])
            {
                int tmp=a[i];
                a[i]=a[j];
                a[j]=tmp;
            }
        }
    }
    cout<<"Sorted array is: "<<endl;
    for (int i = 0; i < n; i++)
    {
        cout<<a[i]<<" ";
    }

    cout<<endl<<"*************************************************************"<<endl;
    cout<<"Program Prepared & Executes by:Muneef Khan, Class Roll no.: 43"<<endl;
    cout<<"*************************************************************"<<endl;
    
    return 0;
}

File: Muneef khan 43 (A-1)/JAVA/Q4.java
//Program to replace all 0's with 1 in a given integer. Given an integer as an input, all the 0's in the number has to be replaced with 1.
import java.util.Scanner; 
class Main
{ 
    public static void main(String args[])
    { 
        Scanner sc = new Scanner(System.in); 
        System.out.print("Enter the number: "); 
        int n=sc.nextInt(); 
        int ans=0; 
        int t=1; 
        while(n>0)
        { 
            int temp=n%10; 
            if(temp==0)
                temp=1; 
            ans+=temp*t; 
            t*=10; 
            n/=10; 
        } 
        System.out.println(ans);

        System.out.println("*************************************************************");
		System.out.println("Program Prepared & Executed by:Muneef Khan,  CLass Roll no.: 43");
		System.out.println("*************************************************************");
        sc.close(); 
    }
}